mw-webapp/src/model/businessModel/StudentComment.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Student comments
+ */
+export class StudentComment {
```

Comment:
Is it one comment or few comments? Mistake in class name or in the documentation

Reply:
Done